deku-flextesa/Dockerfile

```diff
@@ -10,4 +10,4 @@ COPY ./kathmandubox /usr/bin/kathmandubox
 # See contract storage in ../networks/flextesa
 COPY ./deku_flextesa_state.tar /tmp/deku_flextesa_state.tar
 RUN tar -xvf /tmp/deku_flextesa_state.tar -C /tmp
-COPY .tezos-client /root
+COPY .octez-client /root
```

Comment:
```suggestion
+COPY octez-client /root/.octez-client
```

Reply:
Why?

Reply:
[You're changing the name of deku-flextesa/.tezos-client to deku-flextesa/octez-client](https://github.com/marigold-dev/deku/pull/927/files#diff-31616db27e4a4e1461a6eed444c29808dd0397874c01a841d8fdc271b424a527) (mind the `.` in front of the name). I think it's a good idea, but the client expects a `.octez-client`.